powsybl-parent/powsybl-parent-ws/pom.xml

```diff
@@ -68,8 +68,11 @@
             <column name="myuuid" type="UUID">
             still generates
             <modifyDataType columnName="myuuid" newDataType="bytea"/>
+
+            Switching to liquibase-hibernate.version 4.26.0, which includes resolving the issue where it attempted to connect to the database during changelog generation.
+            This connection issue has been corrected in version 4.26.0, along with potential additional bug fixes and enhancements.
```

Comment:
Have you checked the problem explain above?  
If yes, ou can remove fully the comment,.

Reply:
I left the comment to track and explain why there is a difference between the versions of liquibase.core and liquibase.hibernate